frontend/workflows/k8s/src/scale-resources.tsx

```diff
@@ -0,0 +1,289 @@
+
+const DeploymentIdentifier: React.FC<ResolverChild> = ({ resolverType }) => {
+  const { onSubmit } = useWizardContext();
+  const deploymentData = useDataLayout("deploymentData");
+  const inputData = useDataLayout("inputData");
+
+  const onResolve = ({ results, input }) => {
+    // Decide how to process results.
+    deploymentData.assign(results[0]);
+    inputData.assign(input);
+    onSubmit();
+  };
+
+  return <Resolver type={resolverType} searchLimit={1} onResolve={onResolve} />;
+};
+
+const DeploymentDetails: React.FC<WizardChild> = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const deploymentData = useDataLayout("deploymentData");
+  const deployment = deploymentData.displayValue() as IClutch.k8s.v1.Deployment;
+  const update = (key: string, value: boolean) => {
+    deploymentData.updateData(key, value);
+  };
+
+  const currentDeploymentData = useDataLayout("currentDeploymentData");
+
+  const [containerName, setContainerName] = React.useState(
+    deployment.deploymentSpec.template.spec.containers[0].name
+  );
+
+  const [containerIndex, setContainerIndex] = React.useState(0);
+
+  React.useEffect(() => {
+    // save the original values of deployment spec
+    if (deployment) {
+      currentDeploymentData.assign(deployment);
+    }
+  }, []);
+
+  return (
+    <WizardStep error={deploymentData.error} isLoading={deploymentData.isLoading}>
+      <strong>Deployment Details</strong>
+      <MetadataTable
+        onUpdate={update}
+        data={[
+          { name: "Name", value: deployment.name },
+          { name: "Namespace", value: deployment.namespace },
+          {
+            name: "Container Name",
+            value: (
+              <Select
+                label="Container Name"
+                name="containerName"
+                onChange={value => {
+                  setContainerName(value);
+                  setContainerIndex(
+                    deployment.deploymentSpec.template.spec.containers.findIndex(
+                      container => container.name === value
+                    )
+                  );
+                  deploymentData.updateData("containerName", value);
+                }}
+                options={deployment.deploymentSpec.template.spec.containers.map(container => {
+                  return { label: container.name };
+                })}
+              />
+            ),
+          },
+          {
+            name: "CPU Limit",
+            value: deployment.deploymentSpec.template.spec.containers.find(
+              container => container.name === containerName
+            ).resources.limits.cpu,
+            textFieldLabels: {
+              disabledField: "Current Limit",
+              updatedField: "New limit",
+            },
+            input: {
+              type: "string",
+              key: `deploymentSpec.template.spec.containers[${containerIndex}].resources.limits.cpu`,
```

Comment:
We could pull this key out into a var as well since it's used so often.

```
const containerBase = `deploymentSpec.template.spec.containers[${containerIndex}]`;
```

```suggestion
              key: `${containerBase}.resources.limits.cpu`,
```

Reply:
Updated.